packages/neuron-wallet/tests/services/multisig.test.ts

```diff
@@ -237,7 +243,10 @@ describe('multisig service', () => {
       ])
       // @ts-ignore Private method
       await MultisigService.deleteDeadMultisigOutput([defaultMultisigConfig])
-      const multisigOutputs = await getConnection().getRepository(MultisigOutput).createQueryBuilder().getMany()
+      const multisigOutputs = await getConnection()
+        .getRepository(MultisigOutput)
+        .createQueryBuilder()
+        .getMany()
```

Comment:
Are these changes made by linter configured in the project or the one configured locally